content/docs/user-guide/project-structure/configuration.md

```diff
@@ -394,16 +394,35 @@ have no effect.
 ## studio
 
 - `studio.token` - Studio access token to use. When this is set, DVC uses this
-  to notify Studio of new experiments. For security reasons, we advise setting
-  token to either a local or a global config. This can also be specified through
-  `DVC_STUDIO_TOKEN` environment variable.
+  to share [live experiments] and notify Studio about [pushed experiments]. For
+  security reasons, we advise setting token to either a local or a global
+  config. This can also be specified through `DVC_STUDIO_TOKEN` environment
+  variable, which will override any value in `studio.token`.
 
   [Get the token](https://studio.iterative.ai/user/_/profile?section=accessToken)
   or check
   [this guide on how to create an access token](/doc/studio/user-guide/projects-and-experiments/live-metrics-and-plots#set-up-an-access-token).
 
+- `studio.offline` - Disables sharing [live experiments] even if `studio.token`
+  is set or the token has been specified in `DVC_STUDIO_TOKEN`. Offline mode can
+  also be specified through `DVC_STUDIO_OFFLINE` environment variable, which
+  will override any value in `studio.offline`. Accepts values `true` and
+  `false`.
```

Comment:
It actually supports other values, right?

Reply:
Yes, but this is the same as all other bool config options, which are also documented as only accepting `true` and `false`. I think it makes sense to document like this but try to be forgiving internally.